scripts/linters/pylint_extensions.py

```diff
@@ -1917,6 +1917,77 @@ def process_tokens(self, tokens):
                         'single-line-pragma', line=line_num)
 
 
+class SingleSpaceAfterIfElifWhileChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks that there is a single space
+    between an `if`, `elif`, or `while` statement and a bracket `(` if
+    a bracket is used.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'single-space-after-if-elif-while'
+    priority = -1
+    msgs = {
+        # ERIC: Figure out how to name this message. Does 'C0029' make sense?
+        'C0029': (
+            (
+                'A single space is needed between an `if`, `elif`, or `while` '
+                'and the parenthesis.'),
+            'single-space-after-if-elif-while',
+            (
+                'Add a single space between the `if`, `elif`, or `while` '
+                'and the parenthesis.')
+        ),
+    }
+
+    def visit_if_elif(self, node):
```

Comment:
Should we also add this check for [IfExpresions](http://pylint.pycqa.org/projects/astroid/en/latest/api/astroid.nodes.html#astroid.nodes.IfExp)?